docs/remixd.md

```diff
@@ -17,6 +17,14 @@ The code of `remixd` is
 Or just install it in the directory of your choice by removing the -g flag:
 `npm install remixd`
```

Comment:
should be:
npm install -g @remix-project/remixd

Reply:
and please check the live docs:
https://remix-ide.readthedocs.io/en/latest/remixd.html